test/parallel/test-http-set-timeout-server.js

```diff
@@ -169,6 +165,47 @@ test(function idleTimeout(cb) {
       allowHalfOpen: true,
     };
     const c = net.connect(options, () => {
+      c.on('error', (e) => {
+        if (e.message !== 'read ECONNRESET')
+          throw e;
+      });
+      c.write('GET /1 HTTP/1.1\r\nHost: localhost\r\n\r\n');
+      // Keep-Alive
+    });
+  }));
+});
+
+test(function fastTimeout(cb) {
+  let connectionHandlerInvoked = false;
```

Comment:
Comment... (this one tests that timeout callback is fired for the socket but not for `req`)